roles/bind-data-sync-apb/tasks/main.yml

```diff
@@ -2,8 +2,13 @@
   shell: "oc get configmaps/{{ sync_secret_name }} -n {{ namespace }} -o jsonpath={.data.uri}"
   register: sync_server_uri_result
 
+- name: Generate client id
+  shell: tr -d -c "a-zA-Z0-9" < /dev/urandom | head -c 20
```

Comment:
Might it make more sense to use a uuid here? `uuidgen` is a package found on most linux distros. Not sure if it's included in the apb container but it would be worth checking out IMO.

Reply:
We use this pattern in other apbs [1] as well. I've just copy pasted it. If this PR is still valid, should I try the `uuidgen`, or is this approach ok for now?

[1] https://github.com/aerogearcatalog/metrics-apb/pull/78/files#diff-1733a25e041636fec58eb433c685c68fR5